tests/test_pytorch/test_pytorch_functions.py

```diff
@@ -143,6 +143,30 @@ def interpolate_caller(*arg, **kwargs):
     assert np.allclose(model_output, rewrite_output[0], rtol=1e-03, atol=1e-05)
 
 
+@backend_checker(Backend.TENSORRT)
```

Comment:
There is a UT for bicubic interpolation [here](https://github.com/open-mmlab/mmdeploy/blob/f21dc4e7d3ee916edd6d145de3705f748530cd9e/tests/test_ops/test_ops.py#L120)

Reply:
https://github.com/NVIDIA/TensorRT/issues/347
I can't use dynamic_export function because of this issue...
I'm looking for a solution.